test_validation.py

```diff
@@ -111,3 +111,46 @@ def test_validate_size(self):
 
             self.assertEqual(result.is_valid, True)
             self.assertEqual(result.message, "")
+
+
+class TestCustomBoardValidation(unittest.TestCase):
```

Comment:
these tests seem like a good opportunity to try out parameterized tests (they let you test your method against a variety of different params):

https://pypi.org/project/parameterized/

Reply:
Awesome! I'll try it out!